lib/model/card/quest/hispaniola_1_rumours_of_an_island.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:treasure_of_the_high_seas/model/card/action/card_action.dart';
+
+import '../../resource.dart';
+import '../action/discard_action.dart';
+import '../action/exile_action.dart';
+import '../card.dart';
+
+class RumoursOfAnIsland extends Card {
+  const RumoursOfAnIsland() : super("Rumours of an Island");
+
+  @override
+  List<CardAction> getActions() {
+    return [
+      //TODO - flavour text required
+      DiscardAction(""),
```

Comment:
In the doc the flavor text is "Discard this card.". I made flavour text optional for the `DiscardAction`, so if you omit the empty string argument it will default to the right thing

Reply:
Ah, I meant someone needs to come up with some flavour text for the actual card!

Reply:
Ah ok, cool. Still think it's worth omitting the empty string for now so the card has some text (passing an empty string will actively override it to say nothing)

Reply:
Makes sense. Done.